lib/dishes.rb

```diff
@@ -0,0 +1,12 @@
+class Dishes
+
+  attr_reader :menu
+
+  def initialize
+    @menu = {"Seeds" => 50,
+      "Eyeballs" => 25,
+      "Camel Breast" => 40,
+      "shoes" => 78 }
```

Comment:
This is a fantastic idea for a startup. Only peasants go to stores to buy their shoes.